npm install fails with ENOENT

Hi there!

I just noticed a couple of our builds started breaking with the following error:
```bash
     [exec] npm ERR! code ENOENT
     [exec] npm ERR! syscall chmod
     [exec] npm ERR! path /frontend/node_modules/geojson-flatten/geojson-flatten
     [exec] npm ERR! errno -2
     [exec] npm ERR! enoent ENOENT: no such file or directory, chmod '/frontend/node_modules/geojson-flatten/geojson-flatten'
     [exec] npm ERR! enoent This is related to npm not being able to find a file.
     [exec] npm ERR! enoent 
```

Checking out this repository and running:
```bash
$ npm install && npm link
```
also results in the same error.

Removing the `bin` parameter from the `package.json` seems to be the correct fix here so I'll create a PR shortly.

--- a/package.json
+++ b/package.json
@@ -15,9 +15,6 @@
   "directories": {
     "test": "test"
   },
-  "bin": {
-    "geojson-flatten": "./geojson-flatten"
-  },
   "scripts": {
     "test": "eslint index.js && tap test/flatten.js",
     "prepare": "microbundle --name geojsonFlatten",

